pkg/executor/infoschema_reader.go

```diff
@@ -3631,6 +3633,43 @@ func (e *memtableRetriever) setDataForClusterIndexUsage(ctx sessionctx.Context,
 	return nil
```

Comment:
Can you add some tests for it ?

Reply:
It requires a unistore or mock store with keyspace support, which might be out of this pull request scope.